app/lib/irs8812_pdf.rb

```diff
@@ -0,0 +1,44 @@
+class Irs8812Pdf
```

Comment:
I def think its worth namespacing these to a year, though, just to create a precedent moving forward and keep it consistent (even if we don't keep last years PDF for the 1040 around). It would make it more clear in the naming that the implementation is super specific to the tax year. The appearance and content on the 8812 PDF changed a lot from 2020 to 2021 tax years, for example.